test/table-missing-pipes.md

```diff
@@ -0,0 +1,127 @@
+# Table Missing Pipes
+
+## Bad Header Row
+
+| Table | {MD055}
+|-------|---------|
+
+  Table | {MD055} |
+|-------|---------|
+
+  Table | {MD055}
+|-------|---------|
+
+## Bad Separator Row
+
+| Table | Header |
+|-------|---------
+
+{MD055:17}
+
+| Table | Header |
+ -------|--------|
+
+{MD055:22}
+
+| Table | Header |
+ -------|---------
```

Comment:
I notice the missing start pipe is replaced with a space, but the missing end pipe is replaced with a dash - is that significant?

Reply:
Not significant afaik, it may be an artifact from my editor trying to fix the table. Should I change it for consistency?

Reply:
If you don’t mind, please.